pkg/Cpanel/Security/Advisor/Assessors/Kernel.pm

```diff
@@ -64,6 +66,28 @@ sub _check_for_kernel_version {
     elsif ( ( $environment eq 'virtuozzo' ) || ( $environment eq 'lxc' ) ) {
         $self->add_info_advice( 'text' => ['Kernel updates are not supported on this virtualization platform. Be sure to keep the host’s kernel up to date.'] );
```

Comment:
I'm going back and forth on this, but I think that it ultimately should be your call @bk2204  - If we leave this as "info", that means that an email will be sent regarding the state change, which seems a little unnecessary to me. Even in the python script that @bdraco pointed out, this is considered to be an "everything is fine" situation. 

If it weren't for the email notification, I'd say that "info" is the right place for it, but considering the email notifications that are getting sent now, MAYBE "good" might be a better choice. Fewer unnecessary messages is definitely more in line with what I had hoped to accomplish to begin with.

AND SO... I'm leaving the decision in your hands.